### ISSUE TITLE
Add tests

### ISSUE THREAD

**Issue by Fripe070** [ThatOtherAndrew/ThatOtherBot#98]
Unit tests should be added for the parts of breadcord that don't interact with or rely on the discord API.

**Comment by m31-galaxy** [comment]
That is absolutely correct! I have never written tests though and have been kinda intimidated by the idea of doing so, but I'll have to pull myself together and figure something out sooner or later :p
